packages/fxa-settings/src/lib/gql.ts

```diff
@@ -158,7 +158,13 @@ export function createApolloClient(gqlServerUri: string) {
       httpLink,
     ]),
     typeDefs,
-  });
+    connectToDevTools: window.location.host === 'localhost:3030',
+    devtools: {
+      enabled: window.location.host === 'localhost:3030',
+      name: window.location.host === 'localhost:3030' ? 'settings' : '',
+    },
+  };
```

Comment:
Did you mean to leave this in here?

Reply:
I think it's alright. Probably nice for most developers to have. If you add the [dev tools extension](https://www.apollographql.com/docs/react/development-testing/developer-tooling/#apollo-client-devtools), it lets you see the state of the cache which can be pretty helpful.